source/index.md.erb

```diff
@@ -41,1706 +114,2044 @@ curl https://nuv.la?media=xml
+  "resourceURI" : "http://sixsq.com/slipstream/1/ServiceAttributeNamespace",
+  "created" : "2017-04-27T08:41:39.470Z",
+  "updated" : "2017-04-27T08:41:39.470Z",
+
+  "prefix" : "exoscale",
+  "uri" : "http://sixsq.com/slipstream/schema/1/connector/exoscale",
+  
+  "acl" : {"..." : "..."}
+}
+```
 
-### Url Parameters
+Every attribute in a ServiceOffer resource must be namespaced to avoid
+collisions.  The ServiceAttributeNamespace resources maintain the
+mapping between a namespace prefix and the associated, complete
+URI. The parameters are described in the table below.
 
-Parameter | Required | Description
---------- | -------- | -----------
-run-uuid  | true     | Run unique identifier - e.g. *fa3b9652-9dac-49e2-9573-b761777c8238*.
-node-name | true     | Node name - e.g. *centos_node*.
+A ServiceOffer resource cannot be uploaded to the server unless all of
+the namespace prefixes within the document have been defined.
 
-### Body Parameters
+Currently, only an administrator can create, update, or delete
+ServiceAttributeNamespace resources. These actions follow the standard
+CIMI patterns.  Most users will only search these resources and look
+at the details for a particular ServiceAttributeNamespace resource.
 
-Parameter | Required | Description
---------- | -------- | -----------
-ids       | true     | Defines the comma separated list of indices of the node instances to be removed.
+Parameter | Required  | Description
+--------- | --------  | -----------
+prefix | true | namespace prefix
+uri | true | full URI associated with the prefix
 
-## Get instance names of a node in Run
 
-```shell
-curl https://nuv.la/run/cce5ac6d-5465-4773-9875-66f21c65e15e/centos_node -u <user>:<password> "Content-Type: text/plain" -D -
-```
 
-> The above command requests a list of the instance names of the node centos_node.  On success, the return code is 200 and the body contains the comma separated list of the instance names.
+# Monitoring
 
-```http
-HTTP/1.1 200 OK
-Content-type: text/plain; charset=UTF-8
-Content-length: 65
-Server: Restlet-Framework/2.2.1
-Accept-ranges: bytes
-Vary: Accept-Charset, Accept-Encoding, Accept-Language, Accept
-Date: Wed, 25 Jun 2014 21:57:21 GMT
-Set-cookie: com.sixsq.slipstream.cookie=com.sixsq.idtype=local&com.sixsq.identifier=test&com.sixsq.expirydate=1403776630991&com.sixsq.signature=-gkbwu2gknk54fcfbg5di4484bna1tttgn4na0za9fe4apgnkgjhcmc5b4mlqp5yzmnwkyf8meegow8iri4q2cz89a5pm3i7m5d7; Path=/
+## Event (CIMI)
 
-centos_node.1,centos_node.2,centos_node.3
-```
+The resources under `/api` namespace do not support basic authentication and require an authentication cookie.
 
-Get instance names of a node in Run.
+<aside class="notice">
+From
+<a target="_blank" href="http://dmtf.org/sites/default/files/standards/documents/DSP0263_1.0.1.pdf">
+CIMI Model and RESTful HTTP-based Protocol documentation 5.17.13</a>
+<br />
+<br />
+A resource that represents the occurrence of an event within the managed infrastructure.
```

Comment:
Entirely remove this aside and provided a (hopefully) clearer description of the Event resource.